tests/test_headful.py

```diff
@@ -0,0 +1,110 @@
+"""
+Tests relating to headful mode
+"""
+import os
+import platform
+import pytest
+import tempfile
+import shutil
+
+from pyppeteer import launch
+from syncer import sync
+
+EXTENSION_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), "assets", "simple-extension")
+
+
+class TestHeadful:
+    @pytest.fixture
+    def extension_options(self, default_browser_options):
```

Comment:
Regarding my previous comment - I see, you extend these options.
We may do it with wrapping fixture to the closure:
e.g.
```
@pytest.fixture
def browser_with_options():
      def closure_(**options);
            browser = sync(launch(**_launch_options))
            return browser
     return closure_

```
But now I am not sure it's worth it.

Reply:
The reason for fixture is solely to get the default browser options because in some tests I've to change options (ex: from headless to headful mode).